File: projekti-bts/src/main/java/edu/ubt/testimi/entity/Product.java
package edu.ubt.testimi.entity;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter
@Setter
public class Product {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "emri", length = 45, nullable = false)
    private String emri;

    @Column(name = "kodi", length = 45, nullable = false, unique = true)
    private String kodi;

    @Column(columnDefinition = "TEXT")
    private String pershkrimi;

    @Column(nullable = false)
    private int sasia;

    @Column(nullable = false)
    private double cmimi;

    @Column(name = "foto", columnDefinition = "LONGTEXT", nullable = true)
    private String foto;

}

File: projekt-bts-ui/src/app/classes/product/product.ts
export class Product {

  id: Number;
  emri: String;
  kodi: String;
  pershkrimi: String;
  sasia: Number;
  cmimi: Number;
  foto: String;
  eshteNeStok: Boolean;

  constructor(id: Number, emri: String, kodi: String, pershkrimi: String, sasia: Number, cmimi: Number, foto: String, eshteNeStok: Boolean) {
    this.id = id;
    this.emri = emri;
    this.kodi = kodi;
    this.pershkrimi = pershkrimi;
    this.sasia = sasia;
    this.cmimi = cmimi;
    this.foto = foto;
    this.eshteNeStok = eshteNeStok;
  }
}
